### ISSUE TITLE
Support to Elixir

### ISSUE THREAD

**Issue by iautom8things** [lukinco/lukin-vscode-theme#2]
Almost perfect!  So I mostly work in Elixir.  Some themes show a distinction between module names and function names.  Not sure if you could add a second variation, but it'd be great if Lukin could distinguish between them, as well!

<img width="738" alt="Screen Shot 2020-04-03 at 3 35 46 PM" src="https://user-images.githubusercontent.com/522040/78402794-d5d68d80-75c0-11ea-8962-faa91a9340d9.png">

(The other theme is Atelier Cave Dark)

**Comment by diogomoretti** [comment]
Hi @iautom8things, thanks for using our theme.

I'll work on it. Can you give me that code snippet?

**Comment by iautom8things** [comment]
Hi @diogomoretti !  Thank you!

```elixir
@spec reduce(t, (element, acc -> acc)) :: acc
  def reduce(enumerable, fun)

  def reduce([h | t], fun) do
    reduce(t, h, fun)
  end

  def reduce([], _fun) do
    raise Enum.EmptyError
  end

  def reduce(enumerable, fun) do
    Enumerable.reduce(enumerable, {:cont, :first}, fn
      x, {:acc, acc} -> {:cont, {:acc, fun.(x, acc)}}
      x, :first -> {:cont, {:acc, x}}
    end)
    |> elem(1)
    |> case do
      :first -> raise Enum.EmptyError
      {:acc, acc} -> acc
    end
  end

**Comment by diogomoretti** [comment]
Hi @iautom8things, what do you think?

<img width="583" alt="image" src="https://user-images.githubusercontent.com/2853428/78413920-084baf00-75f0-11ea-900b-f3f35a27091a.png">

**Comment by iautom8things** [comment]
Love it!  Thank you :)!

**Comment by diogomoretti** [comment]
Published on `v0.1.5`. Thanks!
